uast/diff/testdata/smalltest.txt

```diff
@@ -0,0 +1,148 @@
+720_1
```

Comment:
Maybe there is a formal definition for this? For example "source size is less than N bytes/lines" or "the UAST size is less than N bytes/nodes"?

Reply:
I'm not sure how I'd chosen these. If there was such definition, how to proceed?

Reply:
Also, Python exists on every modern Linux distro so it's still quite portable (see the alternative: bash)